src/store/wallet/sagas.ts

```diff
@@ -47,26 +57,54 @@ function* sendAsync({
     }
 
     yield put(sendSucceeded())
+    yield put(
+      make(Name.SEND_AUDIO_SUCCESS, {
+        from: account?.wallet,
+        recipient: recipientWallet
+      })
+    )
   } catch (e) {
     yield put(sendFailed({ error: e.message }))
+    yield put(
+      make(Name.SEND_AUDIO_FAILURE, {
+        from: account?.wallet,
+        recipient: recipientWallet
```

Comment:
would it be useful to add the error message here?

Reply:
I dont think we can/should stuff error messages into amplitude, probably more a sentry thing